<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasWin</class>
 <widget class="QWidget" name="CanvasWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LemonxNote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 46, 37);</string>
  </property>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTips">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 150);
border-radius: 15px;
padding: 25px;</string>
   </property>
   <property name="text">
    <string>X: 0, Y: 0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CanvasWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>CanvasWidget</zorder>
  <zorder>labelTime</zorder>
  <zorder>labelTips</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
